src/cpp/statistics/types/typesTypeObjectSupport.cxx

```diff
@@ -67,7 +67,7 @@ void register_EntityId_s_type_identifier(
             ReturnCode_t return_code_value {eprosima::fastdds::dds::RETCODE_OK};
             return_code_value =
                 eprosima::fastdds::dds::DomainParticipantFactory::get_instance()->type_object_registry().get_type_identifiers(
-                "anonymous_array_uint8_t_4", type_ids_value);
+                "anonymous_array_byte_4", type_ids_value);
```

Comment:
Same here

Reply:
This PR only modifies the name the type is registered under. This doesn't affect the interoperability.